Components/Avatar/avatar.css

```diff
@@ -55,9 +57,41 @@ hr {
 .margin-top-1 {
   margin-top: 33rem;
 }
-.avatar {
+
+.avatar-container {
   display: flex;
-  background-color: lightskyblue;
+  min-height: 20rem;
```

Comment:
use color variable.

Reply:
I have changed them to var and structured the code correctly. Thanks for reviewing